warehouse/query-core/src/test/java/datawave/query/attributes/TypeAttributeIT.java

```diff
@@ -30,10 +31,12 @@
  */
 public abstract class TypeAttributeIT {
 
+    private static final Logger log = LoggerFactory.getLogger(TypeAttributeIT.class);
+
     private static final int MAX_ITERATIONS = 100_000;
 
     // normalization context
-    protected static String NORMALIZED = "normalized";
+    protected static String NORMALIZED = "    normalized";
```

Comment:
was the spacing here intended?

Reply:
Yes, this allows you to view pretty A/B numbers in unit tests